### DISCUSSION TITLE
Vercel Deno Function Puppeteer error

### DISCUSSION THREAD

**Discussion by j-xu0** [vercel/community#1340] [Help]
### Summary

After migrating my node.js puppeteer code to Deno, I got the error shown below and it was easily fixed by running `deno run -A --unstable https://deno.land/x/puppeteer@16.2.0/install.ts` in the terminal while testing but after deploying it to Vercel, the error is back and I'm no longer sure how to fix it. 

### Additional information

```
Error: Could not find browser revision 1022525. Run "PUPPETEER_PRODUCT=chrome deno run -A --unstable https://deno.land/x/puppeteer@16.2.0/install.ts" to download a supported browser binary.
at ChromeLauncher.launch (https://deno.land/x/puppeteer@16.2.0/src/deno/Launcher.ts:99:30)
at async scrape (file:///vercel/path0/api/index.ts:24:21)
at async parseScrape (file:///vercel/path0/api/index.ts:55:20)
at async default (file:///vercel/path0/api/index.ts:139:4)
at async processEvents (file:///vercel/path0/.vercel/builders/node_modules/vercel-deno/dist/runtime.ts:191:16)
```

**Comment by TooTallNate** [comment]
`chrome` is not installed within the Serverless Function execution environment. In general, it's not really recommended to run Puppeteer within the Serverless Function environment, but if you can find a Deno URL that packages a chromium binary compiled for the AWS environment then it _might_ work.

**Reply by j-xu0** [reply]
the error has an install.ts file that fixed the issue when I ran the command `deno run -A --unstable https://deno.land/x/puppeteer@16.2.0/install.ts` locally but I don't know how to do this after deployment on Vercel.
